cmd/vic-machine/common/cnetworks.go

```diff
@@ -131,6 +141,22 @@ func parseContainerNetworkDNS(cds []string) (map[string][]net.IP, error) {
 	return dns, nil
 }
 
+func parseContainerNetworkFirewalls(cfs []string) (map[string]executor.TrustLevel, error) {
+	firewalls := make(map[string]executor.TrustLevel)
+	for _, cf := range cfs {
+		vnet, value, err := splitVnetParam(cf)
+		if err != nil {
+			return nil, fmt.Errorf("Error parsing container network parameter %s: %s", cf, err)
+		}
+		trust, err := executor.ConvertStringToTrustLevel(value)
```

Comment:
`ParseTrustLevel` would follow other packages (like `net`) that have similar functions.